dev/Expander/Expander.xaml

```diff
@@ -0,0 +1,204 @@
+                                    </VisualState.Storyboard>
+                                </VisualState>
+                                <VisualState x:Name="Collapsed">
+                                    <VisualState.Storyboard>
+                                        <Storyboard>
+                                            <PopOutThemeAnimation TargetName="ExpanderContent" />
+                                        </Storyboard>
+                                    </VisualState.Storyboard>
+                                </VisualState>
+                            </VisualStateGroup>
+                            <VisualStateGroup x:Name="ExpandDirectionStates">
+                                <VisualState x:Name="Down" />
+                                <VisualState x:Name="Up">
+                                    <VisualState.Setters>
+                                        <Setter Target="ExpanderHeader.(Grid.Row)" Value="1" />
+                                        <Setter Target="ExpanderContent.(Grid.Row)" Value="0" />
+                                        <Setter Target="Row0.Height" Value="*" />
+                                        <Setter Target="Row1.Height" Value="Auto" />
+                                        <Setter Target="PopIn.FromVerticalOffset" Value="50" />
+                                    </VisualState.Setters>
+                                </VisualState>
+                            </VisualStateGroup>
+                        </VisualStateManager.VisualStateGroups>
+
+                        <Grid.RowDefinitions>
+                            <RowDefinition x:Name="Row0" Height="Auto" />
+                            <RowDefinition x:Name="Row1" Height="*" />
+                        </Grid.RowDefinitions>
+                        <Grid.ColumnDefinitions>
+                            <ColumnDefinition x:Name="Column0" Width="*" />
+                            <ColumnDefinition x:Name="Column1" Width="Auto" />
+                        </Grid.ColumnDefinitions>
+
+                        <ToggleButton
+                            x:Name="ExpanderHeader"
+                            AutomationProperties.AutomationId="ExpanderToggleButton"
+                            MinWidth="0"
+                            MinHeight="0"
+                            IsEnabled="{TemplateBinding IsEnabled}"
+                            Style="{StaticResource ExpanderDownHeaderStyle}"
+                            Content="{TemplateBinding Header}"
+                            ContentTemplate="{TemplateBinding HeaderTemplate}"
+                            ContentTemplateSelector="{TemplateBinding HeaderTemplateSelector}"
+                            IsChecked="{Binding Path=IsExpanded, Mode=TwoWay, RelativeSource={RelativeSource TemplatedParent}}"
+                            Foreground="{TemplateBinding Foreground}"
+                            Padding="{TemplateBinding Padding}"
```

Comment:
>Padding [](start = 54, length = 7)

We often template bind the controls padding property to the template part's margin property. Is this correct (it might be I don't know).